### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by morenol**
`headers` is an array

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ See [docs](https://www.fluvio.io/connectors/inbound/http/) here.
...
-| output_type  | text                       | String | `text` = UTF-8 String Output, `json` = UTF-8 JSON Serialized String                        |
-| output_parts | body                       | String | `body` = body only, `full` = all status, header and body parts                             |
+| Option       | default                    | type    | description                                                                                |
+| :------------| :--------------------------| :-----  | :----------------------------------------------------------------------------------------- |
+| interval     | 10s                        | String  | Interval between each HTTP Request. This is in the form of "1s", "10ms", "1m", "1ns", etc. |
+| method       | GET                        | String  | GET, POST, PUT, HEAD                                                                       |
+| endpoint     | -                          | String  | HTTP URL endpoint                                                                          |
+| headers      | -                          | String[]| Request header(s) Key=Value pairs                                                          |
```
Comment (morenol) [inline README.md:19]:
```suggestion
| headers      | -                          | String[]| Request header(s) "Key:Value" pairs                                                        |
```

**Comment by morenol** [comment]
Once we update this, we should update fluvio-website with these changes

**Review by ajhunyady — APPROVED** [review]
Thanks @morenol Looks good!

**Review by galibey — APPROVED** [review]
LGTM

**Pull request merged**